prac_05/word_occurrences.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404 Practical
+Count the number of words in a string
+"""
+
+words_dict = {}
+
+largest_word_length = 0
+
+words = input("Text: ")
+word_list = words.split()
```

Comment:
Generally the argument given to the input function makes a great variable name eg.

text = input("Text: ")
words = text.split()